src/convert.rs

```diff
@@ -22,6 +22,22 @@ pub trait HttpTryFrom<T>: Sized + Sealed {
     fn try_from(t: T) -> Result<Self, Self::Error>;
 }
 
+pub(crate) trait HttpTryInto<T>: Sized {
```

Comment:
Is this added for internal convenience or does it impact the public api somehow.

Reply:
Yea, it was internal convenience, allowing `arg.http_try_into()` instead of `<T as HttpTryFrom<U>>::try_from(arg)`. Since the trait isn't public, and no one can import it, it exposes no public contract.